Tests/sdk_release/create_content_pr.py

```diff
@@ -0,0 +1,151 @@
+    if prs_list:
+        return prs_list[0]
+    return None
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Triggers update-demisto-sdk-version workflow')
+
+    parser.add_argument('-t', '--access_token', help='Github access token', required=True)
+    parser.add_argument('-b', '--release_branch_name', help='The name of the release branch', required=True)
+    parser.add_argument('-r', '--reviewer', help='The reviewer of the pull request', required=True)
+    parser.add_argument('--artifacts-folder', help='Artifacts folder to create the files', required=True)
+    parser.add_argument('-d', '--is_draft', help='Is draft pull request', default='FALSE')
+    options = parser.parse_args()
+    return options
+
+
+def main():
+    install_logging("create_content_pr.log", logger=logging)
+
+    options = options_handler()
+    release_branch_name = options.release_branch_name
+    access_token = options.access_token
+    reviewer = options.reviewer
+    artifacts_folder = options.artifacts_folder
+    is_draft = options.is_draft
+
+    if is_draft and bool(strtobool(is_draft)):
+        is_draft = True
+        logging.info('preparing to trigger update-demisto-sdk-version workflow with draft pull request')
+    else:
+        is_draft = False
+        logging.info('preparing to trigger update-demisto-sdk-version workflow')
+
+    # prepare the inputs for trigger update-demisto-sdk-version workflow
+    inputs = {
+        'reviewer': reviewer,
+        'release_version': release_branch_name,
+        # 'is_draft': is_draft,
+        'release_changes': get_changelog_text(release_branch_name)
+    }
+
+    data = {
+        'ref': 'master',
+        'inputs': inputs
+    }
+
+    headers = {
+        'Content-Type': 'application/vnd.github+json',
+        'Authorization': f'Bearer {access_token}'
+    }
+
+    # trigger update-demisto-sdk-version workflow
+    response = requests.request("POST", UPDATE_SDK_VERSION_WORKFLOW, headers=headers,
+                                data=json.dumps(data), verify=False)
+    if response.status_code != 204:
+        logging.error('Failed to trigger update-demisto-sdk-version workflow')
+        logging.error(response.text)
+        sys.exit(1)
+
+    logging.success('update-demisto-sdk-version workflow triggered successfully')
+
+    logging.info('Waiting for content release pull request creation')
+
+    # initialize timer
+    start = time.time()
+    elapsed: float = 0
+
+    content_pr: dict = {}
+    # wait to content pr to create
+    while not content_pr and elapsed < TIMEOUT:
+        content_pr = get_pr_from_branch('content', release_branch_name, access_token)
+
+        if not content_pr:
+            logging.info('content pull request not created yet')
+
+        time.sleep(60)
+        elapsed = time.time() - start
+
+    if elapsed >= TIMEOUT:
+        logging.error('Timeout reached while waiting for content pull requests creation')
+        sys.exit(1)
```

Comment:
refactor the code, as you might be sleeping for no reason when you do find the PR, and you might have a false-positive when you exit on a timeout and you have found a PR.